src/js/edu-benefits/5490/helpers.jsx

```diff
@@ -8,3 +8,8 @@ export function transform(formConfig, form) {
     }
   });
 }
+
+export const benefitsLabels = {
+  chapter35: 'DEA (Chapter 35)',
+  chapter33: 'Fry scholarship (Chapter 33)'
```

Comment:
Are we going to add links to "Learn More" like we did with the others? I assume we don't have the content for those right now, but is that the plan?